src/lib.rs

```diff
@@ -1638,7 +1638,7 @@ mod tests {
         assert_eq!(Int::from(4) * Int::from(-7), Int::from(-28));
         assert_eq!(Int::from(5) / Int::from(2), Int::from(2));
         assert_eq!(Int::from(9) % Int::from(3), Int::from(0));
-        // assert_eq!(Int::from(3).pow(3), Int::from(27));
+        assert_eq!(Int::from(3).checked_pow(3), Some(Int::from(27)));
```

Comment:
I don't think this fits here, the other assertions aren't for checked operations. Since `.pow` doesn't exist yet, let's just remove this line from the PR.